File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/ActiuniPeProiectReportModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

public class ActiuniPeProiectReportModel {

	private List<ActiuniPeProiectTaskReportModel> actiuniPeProiectTask;
	private List<ActiuniPeProiectRegistruIntrariIesiriReportModel> actiuniPeProiectRegistruIntrariIesiri;
	private List<ActiuniPeProiectNotaCDReportModel> actiuniPeProiectNotaCD;
	
	public List<ActiuniPeProiectTaskReportModel> getActiuniPeProiectTask() {
		return actiuniPeProiectTask;
	}
	
	public void setActiuniPeProiectTask(List<ActiuniPeProiectTaskReportModel> actiuniPeProiectTask) {
		this.actiuniPeProiectTask = actiuniPeProiectTask;
	}
	
	public List<ActiuniPeProiectRegistruIntrariIesiriReportModel> getActiuniPeProiectRegistruIntrariIesiri() {
		return actiuniPeProiectRegistruIntrariIesiri;
	}
	
	public void setActiuniPeProiectRegistruIntrariIesiri(
			List<ActiuniPeProiectRegistruIntrariIesiriReportModel> actiuniPeProiectRegistruIntrariIesiri) {
		this.actiuniPeProiectRegistruIntrariIesiri = actiuniPeProiectRegistruIntrariIesiri;
	}
	
	public List<ActiuniPeProiectNotaCDReportModel> getActiuniPeProiectNotaCD() {
		return actiuniPeProiectNotaCD;
	}
	
	public void setActiuniPeProiectNotaCD(List<ActiuniPeProiectNotaCDReportModel> actiuniPeProiectNotaCD) {
		this.actiuniPeProiectNotaCD = actiuniPeProiectNotaCD;
	}
}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/actiuni-pe-proiect-report.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { ActiuniPeProiectTaskReportModel } from "./actiuni-pe-proiect-task-report.model";
import { ActiuniPeProiectRegistruIntrariIesiriReportModel } from "./actiuni-pe-proiect-registru-intrari-iesiri-report.model";
import { ActiuniPeProiectNotaCDReportModel } from "./actiuni-pe-proiect-nota-cd.model";

@JsonObject
export class ActiuniPeProiectReportModel {

	@JsonProperty("actiuniPeProiectTask", [ActiuniPeProiectTaskReportModel])
	public actiuniPeProiectTask: ActiuniPeProiectTaskReportModel[] = [];

	@JsonProperty("actiuniPeProiectRegistruIntrariIesiri", [ActiuniPeProiectRegistruIntrariIesiriReportModel])
	public actiuniPeProiectRegistruIntrariIesiri: ActiuniPeProiectRegistruIntrariIesiriReportModel[] = [];

	@JsonProperty("actiuniPeProiectNotaCD", [ActiuniPeProiectNotaCDReportModel])
	public actiuniPeProiectNotaCD: ActiuniPeProiectNotaCDReportModel[] = [];
}